network/network_common.go

```diff
@@ -3,9 +3,22 @@ package network
 import (
 	"errors"
 	"net"
+
+	"github.com/DataDog/gohai/utils"
 )
 
-type Network struct{}
+// Network holds network metadata about the host
+type Network struct {
+	// IpAddress is the ipv4 address for the host
+	IpAddress string
+	// IpAddressv6 is the ipv6 address for the host
+	IpAddressv6 string
```

Comment:
nit
```suggestion
	IpAddressV4 string
	// IpAddressV6 is the IPv6 address for the host
	IpAddressV6 string
```

Reply:
Same than https://github.com/DataDog/gohai/pull/100#discussion_r885759254. It's the name used in the `Collect` method.